internal/debug/flags.go

```diff
@@ -242,3 +252,36 @@ func Exit() {
 		closer.Close()
 	}
 }
+
+func validateLogLocation(logLocation string) error {
+	if logLocation == "" {
+		return nil
+	}
```

Comment:
```suggestion
```
This method validated the given path, if you want to special case the empty string, it's better to do it externally like the suggestion I made above